### ISSUE TITLE
Add TimeOverflow app on F-Droid

### ISSUE THREAD

**Issue by sseerrggii** [coopdevs/timeoverflow-mobile-app#10]
from https://github.com/coopdevs/timeoverflow/issues/518

**Comment by gentooza** [comment]
it seems to be possible
https://f-droid.org/2020/10/14/adding-react-native-app-to-f-droid.html

**Comment by sseerrggii** [comment]
Thanks @gentooza 

@sauloperez is trying to update our Android and IOs apps to the last EXPO SDK here https://github.com/sauloperez/my-app after that we can move this changes to this repo. 

I have no idea if it's possible to build the app also to F-Droid, @gentooza could you help with that?

**Comment by gentooza** [comment]
Yes, I guess it's posible, but the app is not licensed just now, as I already asked in #11
If it's not licensed then it's closed source software, so it can't be published in f-droid :-/
